src/p1/header.rs

```diff
@@ -279,6 +279,21 @@ impl Header {
         }
     }
 
+    pub fn strings(&self, name: &str) -> Vec<&str> {
+        let mut strings = vec![];
+        for (_, k, v) in self.0.iter() {
+            if k.starts_with('/') {
+                continue;
+            }
+            if k == name {
+                for v in v.split(',') {
+                    strings.push(v.trim());
+                }
```

Comment:
Nopes is wrong behaviour based on function name. `.string()` and `.strings()` do completely different things. `.strings()` should not do comma separated behaviour and return all headers matching a given name, and the called then can do a further csv handling.